### PR TITLE
fix some spelling

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs
Comment (sharwell) [inline test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: 📝 I would prefer to wait until #122 is merged to include the test changes.

Comment (SimonCropp) [reply test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: @sharwell no worries. let me know when u want it re-submitted

Comment (sharwell) [reply test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: My change is now merged 👍

**Review by sharwell — APPROVED** [review]

**Comment by bradwilson** [comment]
Thanks!

**Pull request merged**
